examples/minio/slack/README.md

```diff
@@ -1,7 +1,25 @@
 # Send messages on objects upload to a bucket
 
-This function is triggered when an object is added to a bucket
+This function is triggered when an object is added to a bucket and sends an Slack message
+
+You need a SLACK_API_TOKEN
```

Comment:
it'd better to add this line to direct reader to the slack api token page as Daniel's suggestion: 

https://api.slack.com/custom-integrations/legacy-tokens